samples/apps/copilot-chat-app/importdocument/README.md

```diff
@@ -28,16 +28,19 @@ specific contexts that are generally unavailable publicly, such as enterprise da
    `RedirectUri` is the Redirect URI also from the app registration in the Azure Portal, and
    `ServiceUri` is the address the web api is running at.
 
-   > `ClientId` and `RedirectUri` are optional if you only want to ingest documents to the global collection.
+   > `ClientId` and `RedirectUri` are optional if you only want to import documents to the global collection.
 3. Change directory to this folder root.
-4. **Run** the following command to ingest a document to the app under the global document collection where
+4. **Run** the following command to import a document to the app under the global document collection where
    all users will have access to:
    `dotnet run -- --file .\sample-docs\ms10k.txt`
    
-   Or **Run** the following command to ingest a document to the app under a user isolated document collection where
+   Or **Run** the following command to import a document to the app under a user isolated document collection where
    only the user who uploaded the document will have access to:
    `dotnet run -- --file .\sample-docs\ms10k.txt --user-collection`
 
    > Note that this will open a browser window for you to sign in to retrieve your user id. 
 
-   > Currently only supports txt files. A sample file is provided under ./sample-docs.
\ No newline at end of file
+   > Currently only supports txt files. A sample file is provided under ./sample-docs.
+
+   It will take a while for the service to import a document as it needs to invoke the embedding model multiple times.
```

Comment:
Importing may take some time to generate embeddings for each piece/chunk of a document.